src/util.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @fileoverview Utility functions for ESLint CSS plugin.
+ * @author Nicholas C. Zakas
+ */
+
+//-----------------------------------------------------------------------------
+// Type Definitions
+//-----------------------------------------------------------------------------
+
+/*
+ * Note: Using `import()` in the JSDoc comments below because including them as
+ * typedef comments here caused Rollup to remove them. I couldn't figure out why
+ * this was happening so just working around for now.
+ */
```

Comment:
It could be a bug in tree-shaking. When I remove `function isSyntaxReferenceError` (which is unused and gets removed by tree-shaking either way), Rollup doesn't remove `/** @typedef {import("css-tree").SyntaxMatchError} SyntaxMatchError */`.

Reply:
Ooh interesting. Let me look at that.